### PR TITLE
docs: convert examples to standalone

### PR DESCRIPTION
**Pull request by crisbeto**
Converts all of the Components examples to standalone to reduce the amount of boilerplate we need to maintain and to simplify the Stackblitz.

### CODE REVIEW

File: tools/example-module/generate-example-module.ts [hunk 1]
```diff
@@ -51,21 +43,24 @@ function inlineExampleModuleTemplate(parsedData: AnalyzedExamples): string {
...
 
     return result;
   }, {} as any);
 
+  // console.log(exampleMetadata);
+
+  // if (Date.now()) {
+  //   throw new Error('oh no');
+  // }
+
```
Comment (devversion) [inline tools/example-module/generate-example-module.ts:57]: leftover from debugging

File: tools/example-module/example-module.template [hunk 2]
```diff
@@ -19,19 +19,8 @@ export interface LiveExample {
   packagePath: string;
```
Comment (devversion) [inline tools/example-module/example-module.template:19]: I guess at some point we should clean up the file name then.

**Review by devversion — APPROVED** [review]
LGTM, just one minor comment. 

Note that I didn't fully check every example and their imports (just to capture this)

**Comment by jeffschulzusc** [comment]
hi ...

do you know what is status of:
https://github.com/angular/components/tree/main/src/components-examples/material-experimental/column-resize/opt-in-column-resize

i gave it a try but it seems does not work ... i don't see any col-resize cursor or other on my table columns

thanks much

**Pull request merged**